management/telemetry/telemetry-dashboards/telemetry-dashboards.md

```diff
@@ -0,0 +1,197 @@
+{: .image-popup}
+![Screenshot - Configuration Health](/management/telemetry/telemetry-dashboards/ou-config-health.png)
+
+*Configurations Health* gives the health of flows and jobs over time. By hovering over a day, you can see more detailed data (the numbers of flows or jobs on that day that ended with success, error, 
+or warning).
+
+## Activity Center
+*Activity Center* provides the most detailed insight into telemetry data from the three modes available. It is not available by default but has to be requested. You need to tell your account 
+manager which project you want to be the target project in your Activity Center. You can have only one target project. Once the Activity Center has been enabled, it will be visible to everyone with access
+to the target project.
+
+{: .image-popup}
+![Screenshot - Activity Centre in KBC](/management/telemetry/telemetry-dashboards/activity-centre-kbc.png)
+
+The *Activity Center* has five tabs, each focused on a different aspect of consumption monitoring. These tabs are available by default for all projects in the organization but can be filtered for specific 
+ones. You can allows filter by date. The default setup is for the current month and the two previous months. For example, if you access the *Activity Center* during May, you will get data from 1st March until 
+today. However, *Jobs Monitoring* provides detailed information on individual jobs, so it shows only yesterday's data by default. 
+
+All these filters can be changed. For some visuals, you can drill down to see a lower level of detail. For a  graphical representation, the drill-down can be accessed by hovering over a point. The drill-down 
+is available for a number if it has an underline. When you hover ove a visual, a question mark appears in the top right corner. If you click on the question mark, you will get a description of the visual 
+(e.g., which filters are applicable and the logic behind the calculation).
+
+{: .image-popup}
+![Screenshot - Activity Centre Visual Description](/management/telemetry/telemetry-dashboards/ac-info-button.png)
+
+### Organization Usage Activity Center
+This tab provides a high-level overview of the consumption within the entire organization and in relation to contracts.
+It focuses on five different sections: *KPIs*, *Active Contract Consumption*, *Consumption*, *Projects*, and *Configuration Health*. 
+
+#### KPIs
+{: .image-popup}
+![Screenshot - KPIs](/management/telemetry/telemetry-dashboards/ac-org-usage-kpis.png)
+
+*KPIs* show four of the metrics specified in the contract in comparison to the limits. For *PPU*, the usage and limit are shown for the period selected in the date range filter. For the others (*Projects*, 
+*Users*, and *Storage TB*), the values are for the current month only, and the limits are as defined in the contract. 
+
+#### Active Contract Consumption
+{: .image-popup}
+![Screenshot - KPIs](/management/telemetry/telemetry-dashboards/ac-org-usage-contract.png)
```

Comment:
Incorrectly named screenshot - instead of KPIs should be Active Contract Consumption